website/docusaurus.config.js

```diff
@@ -155,7 +155,7 @@ const config = {
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} ChipsAlliance. Built with Docusaurus.`,
+        copyright: `Copyright © Chisel a Series of LF Projects, LLC For web site terms of use, trademark policy and other project policies please see https://lfprojects.org. Built with Docusaurus.`,
```

Comment:
```suggestion
        copyright: `Copyright © Chisel a Series of LF Projects, LLC. For web site terms of use, trademark policy and other project policies please see https://lfprojects.org.`,
```
Is it okay if we add a period between `LLC` and `For`?

Also we can drop the "Built with Docusaurus", this is getting a bit long so we might as well just keep it to the actual copyright.